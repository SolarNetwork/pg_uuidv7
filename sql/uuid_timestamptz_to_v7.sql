SELECT uuid_timestamptz_to_v7('2023-09-12 11:22:33.123456789Z'::timestamptz, TRUE);
