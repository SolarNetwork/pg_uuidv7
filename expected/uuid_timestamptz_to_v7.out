SELECT uuid_timestamptz_to_v7('2023-09-12 11:22:33.123456789Z'::timestamptz, TRUE);
        uuid_timestamptz_to_v7        
--------------------------------------
 018a8920-f523-7000-8000-000000000000
(1 row)

